src/index.js

```diff
@@ -32,7 +32,7 @@ const getQuery = (props) =>
 
 const createMatchMedia = (props, query) => {
   const values = getValues(props)
-  const forceStatic = values.length === 0
+  const forceStatic = Object.keys(values).length === 0
```

Comment:
Seems funky here, we never pass the `values` props on client side
https://github.com/contra/react-responsive#using-css-media-queries which now force all client side using a static match instead of media query

https://github.com/ncochard/matchmediaquery/blob/master/index.js#L16

Reply:
not the best solution but seems like we should only forceStatic when `props.values` is `undefined` and `Object.keys(values).length === 0`

Reply:
@roderickhsiao @Herdismaria Sorry if I didn't test this thoroughly - I don't use SSR so things like this will slip by me.

Does inverting `Object.keys(values).length === 0` to `Object.keys(values).length !== 0` fix it for you?

Reply:
I would love to have some better tests around this behavior to prevent regressions in the future ^

Reply:
Yes it works :)

Reply:
@roderickhsiao 	Thought so, I'll publish a 6.1.1 now to fix it

Reply:
6.1.1 published and 6.1.0 deprecated, thanks for the heads up @roderickhsiao